docs/reference/semantic-segmentation.md

```diff
@@ -0,0 +1,153 @@
+## semantic_segmentation module
+
+The *semantic segmentation* module contains the *BisenetLearner* class, which inherit from the abstract class *Learner*.
+
+
+### Class BisenetLearner
+Bases: `engine.learners.Learner`
+
+The *BisenetLearner* class is a wrapper of the BiseNet model [[1]](#bisenetp) found on [BiseNet] (https://github.com/ooooverflow/BiSeNet).
+It is used to train Semantic Segmentation models on RGB images and run inference.
+
+
+
+
+
+The [BisenetLearner](#src.opendr.perception.semantic_segmentation.bisenet.bisenet_learner.py) class has the
+following public methods:
+
+#### `BisenetLearner` constructor
+```python
+BisenetLearner(self, lr, iters, batch_size, optimizer, checkpoint_after_iter, checkpoint_load_iter, val_after, temp_path, device, loss, weight_decay, momentum, drop_last, pin_memory, num_workers, num_classes)
```

Comment:
Could please double check this documentation?
The arguments doesn't match the ones in the code:
* some arguments are missing
* some arguments are not listed in the correct order (here and below)
* some default values are wrong